Reports from various sites show that this test is unreliable.
As it is not critical just skip it.

@Author: Alexander Grund (TU Dresden)

diff --git a/test/test_jit.py b/test/test_jit.py
index 2cdc5413dfe..5b145458630 100644
--- a/test/test_jit.py
+++ b/test/test_jit.py
@@ -12838,6 +12838,7 @@ dedent """
             data = reader.get_record(str(offset))
             assert(data == buffers[i])
 
+    @unittest.skip("The memory allocation tracking is not reliable on all systems")
     def test_file_reader_no_memory_leak(self):
         num_iters = 10000
         filename, _, _ = self._make_filereader_test_file()
